src/index.js

```diff
@@ -1,25 +1,32 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
-import { Provider } from 'react-redux'
-import configureStore from './store';
+import { createStore, applyMiddleware } from 'redux';
+import thunk from 'redux-thunk';
+import { Provider } from 'react-redux';
 import { BrowserRouter as Router } from "react-router-dom";
 import { Route } from 'react-router-dom';
 import './index.css';
-import * as serviceWorker from './serviceWorker';
+
 import '../node_modules/bootstrap/dist/css/bootstrap.min.css';
```

Comment:
Is this required here?

Reply:
ServiceWorker is invoked by default through create-react-app, but, because we do not need to cache anything asynchronously, it's not really required in the current application. I have removed it. @abhicool957